gpu/cmd/webgpuinfo/README.md

```diff
@@ -0,0 +1,8 @@
+# webgpuinfo
+
+This tool prints out information about your available WebGPU devices.  Do this:
```

Comment:
Instead of this README, add a package comment of the form `Command webgpuinfo prints out...`. In general, we prefer package comments over READMEs when there is not much information, and we *never* create both a README and a package comment with the exact same information.

Reply:
I for one always browse things on github where you see the README and NOT the doc comment.